.github/workflows/projectapollo.yml

```diff
@@ -0,0 +1,17 @@
+name: Project Apollo push build
+
+on: [push]
+
+jobs:
+  build:
+    name: Build on Ubuntu 16.04 with dotnet 3.1.100
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '3.1.100'
+    - name: Build with dotnet
+      run: dotnet build --configuration Release
```

Comment:
Should perform: dotnet restore
Then perform: dotnet build --configuration Release

Reply:
I'm unsure why but github's actions don't always auto perform restore so its not 100% reliable to just call build.

Reply:
Added as requested, and good to know!